drivers/include/bmx055.h

```diff
@@ -36,17 +36,36 @@ extern "C" {
 #endif
 
 /**
- * @name   The sensors default I2C addresses
+ * @defgroup drivers_bmx055_config     BMX055 sensor driver compile configuration
```

Comment:
I think adding the parameters to the config group and adding the `#ifndef` could be done in one commit.

Reply:
So should I squash the two commits ?

Reply:
Yes, please. But do so only when the review is finished

Reply:
Okay